sources/assets/exegol/load_supported_setups.sh

```diff
@@ -149,24 +172,23 @@ function deploy_firefox_addons() {
     else
       cp --preserve=mode /.exegol/skel/firefox/addons.txt "$MY_SETUP_PATH/firefox/addons.txt"
     fi
-    # shellcheck disable=SC2086
-    python3 /opt/tools/firefox/user-setup.py -L $addon_list -D $addon_folder
+    python3 /opt/tools/firefox/user-setup.py -L "$addon_list" -D "$addon_folder"
   else
     mkdir --parents "$MY_SETUP_PATH/firefox/addons" && chmod 770 -R "$MY_SETUP_PATH/firefox/addons"
     cp --preserve=mode /.exegol/skel/firefox/addons.txt "$MY_SETUP_PATH/firefox/addons.txt"
   fi
 }
 
 function deploy_bloodhound_config() {
+  colorecho "Deploying BloodHound User Config"
   [[ -f "$my_setup_bh_path/config.json" ]] && cp "$my_setup_bh_path/config.json" "$bh_config_homedir/config.json"
 }
 
 function deploy_bloodhound_customqueries_merge() {
+  colorecho "Deploying BloodHound Custom Queries"
```

Comment:
I think it should be "Merging with BloodHound Exegol Custom Queries" to be consistent with line 200.

Reply:
Indeed, thank you! Fixed in https://github.com/ThePorgs/Exegol-images/pull/246/commits/4e2d64314788f6e317f944211dff3709a79c8071